script.js

```diff
@@ -24,7 +24,18 @@ jQuery(function() {
 function enforceSummary() {
     var $summary = jQuery('#edit__summary'); // get summary field
     var $minoredit = jQuery('#minoredit');
-    var $savebutton = jQuery("#edbtn__save");
+    var ckgd =  jQuery( "input[type=checkbox][name=ckgdoku]:checked" ).val();
+    var ckge =  jQuery( "input[type=checkbox][name=ckgedit]:checked" ).val();
+    var m_class;
+    if(typeof ckgd == 'string' || typeof ckge == 'string') {       
+        var $savebutton = jQuery("#save_button");
+        m_class = "plugin_enforcesummary_missing";
+    }
+    else {
+        $savebutton = jQuery("#edbtn__save");
```

Comment:
This declares a global variable as far as I'm aware. I think `$savebutton` should be declared before the `if`-statement.